include/libunwind-aarch64.h

```diff
@@ -34,6 +34,11 @@ extern "C" {
 #include <inttypes.h>
 #include <stddef.h>
 #include <ucontext.h>
+#include "compiler.h"
+
+#ifndef UNW_EMPTY_STRUCT
```

Comment:
So with this patch along the user is supposed to define this to something that works?

That sounds fine for now, just to make sure I understand.

Reply:
Yes.  This is what I was using.
```cmake
    add_definitions("-DUNW_EMPTY_STRUCT=char unused\;")
```
We could of course make it simpler somehow.

Reply:
Any reason we can't just define this to char unused; right here for MSVC?